### ISSUE TITLE
New self-synonymized taxa in Systema Dipterorum

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#627]
3.334 provisionally accepted taxa in Systema Dipterorum have a self-synonymizing synonym record. All of these taxa seem to be new in Feb 2024 version.

Some examples:
Agromyza dimidiata	Melander, 1913
Agromyza invaria	Walker, 1857
Agromyza varipes	Macquart, 1835
Phytomyza chloophila	Hering, 1951
Phytomyza diminuta	Walker, 1858
Phytomyza solita	Walker, 1858
Anthomya gagatea	Robineau-Desvoidy, 1830

**Comment by yroskov** [comment]
Thank you, @aoern!

**Comment by yroskov** [comment]
Unfortunately, I do not see these cases through TASK reports. I guess, CLB considers these causes ([ACC-SYN species (same accepted, same authors) 1 of 1](https://www.checklistbank.org/catalogue/3/dataset/1101/duplicates?_colCheck=c7)) as resolved because accepted name is already flagged as "provisionally accepted". It is fine for me. What do you think, @mdoering? 

![image](https://github.com/CatalogueOfLife/data/assets/28574252/b25e14d4-3fbe-44c7-9425-09a3e1472483)

**Comment by mdoering** [comment]
I don't see any sense in keeping a synonym which is identical to its accepted name. In fact I remove them in the extended catalogue. Is there a reason to keep them?

**Comment by proceps** [comment]
There is no combination Agromyza dimidiata in TW. This should be an artefact of the data migration from TW to COL.

**Comment by proceps** [comment]
![image](https://github.com/CatalogueOfLife/data/assets/5597748/20d4811c-4777-4149-95cf-fffb60eb9e37)

**Comment by mdoering** [comment]
Looking for duplicates with the same author, rank and accepted name gives me the list:
https://www.checklistbank.org/dataset/1101/duplicates?acceptedDifferent=false&authorshipDifferent=false&category=binomial&limit=50&rankDifferent=false

**Comment by yroskov** [comment]
That report is inconsistent with TASK report on duplicates in Systema Dipterorum INSIDE the project: https://www.checklistbank.org/catalogue/3/dataset/1101/duplicates?acceptedDifferent=false&authorshipDifferent=false&catalogueKey=3&category=binomial&limit=50&minSize=2&mode=STRICT&offset=0&status=accepted&status=synonym

**Comment by yroskov** [comment]
> There is no combination Agromyza dimidiata in TW. This should be an artefact of the data migration from TW to COL.

Yes, CLB it's doing as implemented procedure: it removes square brackets from genus and applies CoL "provisionally accepted" status.

**Comment by proceps** [comment]
I checked few names. Some of them are real (homonymic names), and some have the same issue with the previous one. Additional combination is added during TW to CoL migration.
